R/uploadOMLTask.R

```diff
@@ -0,0 +1,24 @@
+#' @title Upload a task to the OpenML server.
+#'
+#' @description
+#' Share a task by uploading it to the OpenML server.
+#'
+#' @export
+uploadOMLTask = function(task_type_id, source_data, target_feature, estimation_procedure,
```

Comment:
instead of task_type_id it is more user friendly to use `task.type` like in the `listOMLTasks` functions (see also `listOMLTaskTypes` to get the mapping from the name to the id).